.github/workflows/package.yaml

```diff
@@ -0,0 +1,55 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'v*'
+
+env:
+  REGISTRY: ghcr.io
```

Comment:
Why are you using this registry instead of our company DockerHub? (I get the point of using that as a "middle" registry, but I think this is not the case)

Reply:
> Why are you using this registry instead of our company DockerHub? 

In this case, the main reason is that I still don't have access myself to the DockerHub (waiting on invite to the 1pass vault) so I just wanted to have something working.

> (I get the point of using that as a "middle" registry, but I think this is not the case)

I actually don't consider the github registry a "middle" registry but a fully-fledged alternative to DockerHub. 

This is just my personal opinion but I actually like the github registry better because:

- Everything is centralized in the GitHub repository, no need to access other platform (even for visibility you can see the package(s) in the repo overview)
- Access control via github users is way nicer for non-public things (although this doesn't apply here)

Anyway, happy to push to DockerHub if that is what you need on your side

Reply:
Don't get me wrong, I'm not blaming GitHub registry, it's a perfectly valid option. What I mean is, knowing that our organization is configured and based in Docker Hub (and that's not going to change, at least in the short term), I see the need of using a middle registry for certain purposes in some actions but not in this case, as I mentioned in my firs comment. In this case, it doesn't matter which registry you use (as long as it is private and secure enough).

Yeah, change it to DockerHub, next time you face a similar problem, ping me directly, and I will do the needed part if you don't have access yet.

Reply:
Updated!